### PR TITLE
eliminated bad css styles on Ansaetze page

### LINKED ISSUE #110: revise CSS files
# Bounty

## Scope

- technical issue - make css specific to class on a specific page

## Deliverables
- add style to Vergleich Page
- changes to Ansaetze.css or others
- make sure it does not break other pages

## Gain for the project

- Better appearance
- less style conflict

## Roles

bounty worker: @tratond01 
bounty reviewer: @username

## Issue creation checklist

- [x] I have attached one of the team labels `Fragebogen`, `Technik`, `Projektwebseite`, `Vergleich`, `Organisation` to this bounty
- [x] I have attached one of the priority labels `Blocker`, `High`, `Medium`, `Low` to this bounty
- [x] I have attached one of the `size-` labels to this bounty
- [x] I have added this bounty to bounty board here https://github.com/FUB-HCC/20-SWP-CodingOpenness/projects/1

### PR DESCRIPTION
**Pull request by tratond01**
Please remember the / Bitte beachte die [Contribution Guidelines](https://github.com/FUB-HCC/20-SWP-CodingOpenness/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

- cleans up css on Ansaetze page, so that it does not conflict with other files. 

## Recommendations for testing

New style needed for Vergleich page, previous css styles are commented out in Ansaetze.css for an easier creation of a css file for Vergleich page. 

## Links to relevant issues or information

Closes #110

### CODE REVIEW

**Comment by omerhod** [comment]
plz delete commented code

File: src/Components/Ansaetze/Ansaetze.css [hunk 1]
```diff
@@ -60,10 +52,10 @@ h4 {
   color: white;
 }
 
-.overview-table {
+/*.overview-table {
```
Comment (omerhod) [inline src/Components/Ansaetze/Ansaetze.css]: delete if not used anymore

File: src/Components/Ansaetze/Ansaetze.css [hunk 2]
```diff
@@ -110,7 +102,7 @@ h4 {
   padding: 50px;
 }
 
-table {
+/*table {
```
Comment (omerhod) [inline src/Components/Ansaetze/Ansaetze.css]: also - delete if not needed

Comment (tratond01) [reply src/Components/Ansaetze/Ansaetze.css]: follow-up commits added

**Pull request closed without merging**